Assets/Scripts/Player/Input Parser.cs

```diff
@@ -0,0 +1,67 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+using FrameWork;
```

Comment:
```suggestion
using UnityEngine;
using UnityEngine.InputSystem;

using FrameWork;
```
[Code convention](https://github.com/Team-Swamp/Codename-BDLMTW/wiki/Code-convention#importing-namespaces)